### PR TITLE
First `README.md` file skeleton

### PR DESCRIPTION
**Pull request by andrsj**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+## Table of Contents
+
+- [Ryokai](#ryokai)
+  - [Intro](#intro)
+  - [Table of Contents](#table-of-contents)
+  - [Getting started](#getting-started)
+    - [Prerequisites](#prerequisites)
+    - [Installation](#installation)
+  - [Usage](#usage)
+  - [Roadmap](#roadmap)
```
Comment (MrLutik) [inline README.md]: Nope

File: README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# Ryokai
+
+## Intro
+
+`Ryokai` is the **new infrastructure manager tool** from [KiraCore team](https://github.com/KiraCore/).
+
+> `RYOKAI` supports network operations by automating on-chain and off-chain upgrades, scaling, RollApp deployment, simplifying consensus nodes' management tasks, and enhancing the network's efficiency and security. In other words, `RYOKAI` is an all-in-one DevOps tool that turns any network participant into an infrastructure expert without the need for years of experience.
```
Comment (MrLutik) [inline README.md]: Add the vault integration paragraph with the link.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
+# Ryokai
+
+## Intro
+
+`Ryokai` is the **new infrastructure manager tool** from [KiraCore team](https://github.com/KiraCore/).
```
Comment (MrLutik) [inline README.md]: ** new orchestrator tool***

**Review by MrLutik — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (andrsj) [reply README.md]: @MrLutik shall we not have a roadmap?

**Review by MrLutik — APPROVED** [review]

**Review by PeepoFrog — APPROVED** [review]

**Pull request merged**